Umbraco-Cloud/Set-Up/Manage-CDN-Caching/index.md

```diff
@@ -0,0 +1,128 @@
+---
+versionFrom: 7.0.0
+---
+
+# Managing CDN Caching and Optimizations
+
+Once you have added your own hostnames to your Umbraco Cloud project it's possible to configure certain CDN Caching and minification options for all or for specific hostnames within your project.
+These caching options all relate to the traffic that goes through your hostname from the origin (Umbraco Cloud) to the end-user - meaning the traffic your website and assets from the webserver to the browser.
+
+The options that are currently available are
+- Enable Caching (default: off)
+- Cache TTL (default: 120 minutes)
+- Cache Everything (default: off)
+- Minify Css (default: off)
+- Minify Javascript (default: off)
+- Minify HTML (default: off)
+
+When a new hostname is added to a Project it will have the default settings applied. But you can change the defaults for your Project, so new hostnames will get the default settings you have chosen. This also means that if you enable caching in the default settings and add a new hostname, that caching will be enabled for that new hostname.
```

Comment:
```suggestion
When a new hostname is added to a Project it will have the default settings applied. However you can change the default settings for your project, so new hostnames will get the settings you have chosen. This also means that if you enable caching in the default settings and add a new hostname, that caching will be enabled for that new hostname.
```